### ISSUE TITLE
Unable to Install Screen Reader

### ISSUE THREAD

**Issue by dangero2000** [fbacher/service.kodi.tts#5]
Hello,
Unfortunately, I haven't been able to install this version of the screen reader. Whenever I use the install From ZIP File option, it fails. If I try manually extracting it to the addon directory, it also fails.
I have an alternative version of the screen reader that's slightly older. Do I have to delete both the original addon and the existing user data folder? I thought I just had to delete the addon folder.

**Comment by fbacher** [comment]
In a few days I should have an alpha release, with tested instructions on how to install on Windows and Linux.

**Comment by fbacher** [comment]
There is now an alpha release ready for testing. SETUP.md gives instructions. Please let me know if you are interested. I am able to give much better attention to this now.

Currently:

  * Linux supports eSpeak and Google TTS along with the audio players mpv & mplayer
  * Windows supports eSpeak, Google TTs and Navigator along with the mpv audio player

**Comment by fbacher** [comment]
I finally realized that this defect is a duplicate of issue #3
